fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/service/ReadReportingService.java

```diff
@@ -44,4 +47,7 @@
     ReportData retrieveReport(final Long id);
 
     Collection<String> getAllowedReportTypes();
+    
+    ByteArrayOutputStream generatePentahoReportAsOutputStream(String reportName, String outputTypeParam,
```

Comment:
I don't see any reference to this method. Is it required as implementation returns just null